internal/pkg/pkg.go

```diff
@@ -0,0 +1,53 @@
+// Package pkg defines methods and heuristics for sanitizing package output.
+package pkg
+
+import (
+	"github.com/replit/upm/internal/api"
+)
+
+func makeLoweredHM(b api.LanguageBackend, names []string) map[api.PkgName]struct{} {
+	// Build a hashset. struct{}{} purportedly is of size 0, so this is as good as we get
+	set := make(map[api.PkgName]struct{})
+	for _, pkg := range names {
+		normal := b.NormalizePackageName(api.PkgName(pkg))
```

Comment:
In my research, a map of empty structures is preferred to a map of `bool` because bool takes memory where the empty structure consumes zero bytes. Inclusion in the hashset is sufficient, I don't need an additional boolean

Reply:
Resolved over slack! I didn't know about default values in Go, relying on that default value makes this read much cleaner.